modules/indexExchangeBidAdapter.js

```diff
@@ -419,6 +426,9 @@ var IndexExchangeAdapter = function IndexExchangeAdapter() {
     _IndexRequestData.targetAggregate = {'open': {}, 'private': {}};
 
     if (!utils.hasValidBidRequest(bidArr[0].params, requiredParams, ADAPTER_NAME)) {
+      for (var i = 0; i < bidArr.length; i++) {
+        passOnBid(bidArr[i].placementCode);
+      }
```

Comment:
I second you on this: `bidRequested` is called for non valid sizes.

Say I call this adapter with `[[728, 90], [1000, 90]]`, the size `1000x90` is invalid, but I'll get two calls to event `bidRequested`, when I would expect only one for the size of the adUnit that is accepted by IX.